core/lsp/helpers.cc

```diff
@@ -0,0 +1,189 @@
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_join.h"
+#include "absl/strings/str_split.h"
+#include "core/core.h"
+
+using namespace std;
+
+namespace sorbet::core::lsp {
+
+core::TypePtr getResultType(const core::GlobalState &gs, const core::TypePtr &type, core::SymbolRef inWhat,
```

Comment:
The two methods that I've moved to this file are largely [duplicated in `packager/rbi_gen.cc`](https://github.com/sorbet/sorbet/blob/ca78893d96fbe7c4811f9ccb0fd9f8ddc24dfac3/packager/rbi_gen.cc#L86-L89), albeit with subtle modifications. I suspect we will want to unify these implementations at some point, but I did not do it here because it is tangential to my goal. However, might it make sense to move these to a different location in `core/` so that it is less awkward to use these helpers from code that is not related to the LSP?

Reply:
> might it make sense to move these to a different location in core/ so that it is less awkward to use these helpers from code that is not related to the LSP?

This was addressed in 55ccd46.

> I suspect we will want to unify these implementations at some point, but I did not do it here because it is tangential to my goal

This is still unaddressed, but I'm resolving this comment nonetheless since, as stated above, I do not intend to address that in the PR>